pynalyser/normalize_ast.py

```diff
@@ -31,4 +31,4 @@ def visit_Constant(self, node: ast.Constant) -> ast.Constant:
 
 
 def normalize_ast(tree: ast.AST) -> ast.AST:
-    return ast.fix_missing_locations(AstNormalizer().visit(tree))
+    return AstNormalizer().visit(tree)
```

Comment:
What is the reason for removing this function?

Reply:
you asked me to do this...

Reply:
hmph ok